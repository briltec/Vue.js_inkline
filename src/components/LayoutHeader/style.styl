@import '../../resources/css/config'
@import '_variables'


.layout-header
    flex: 0 0 auto
    background: --layout-header-background
    padding: --layout-header-padding

