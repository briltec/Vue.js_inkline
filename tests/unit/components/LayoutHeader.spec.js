import { shallowMount } from '@vue/test-utils';
import LayoutHeader from 'inkline/components/LayoutHeader';

describe('Components', () => {
    describe('LayoutHeader', () => {
        let wrapper;

        beforeEach(() => {
            wrapper = shallowMount(LayoutHeader);
        });

        it('should render correctly', () => {
            expect(wrapper.html()).toMatchSnapshot();
        });
    });
});
