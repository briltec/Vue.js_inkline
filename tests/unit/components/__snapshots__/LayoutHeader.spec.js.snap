// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components LayoutHeader should render correctly 1`] = `
<button tag="button" tabindex="1" class="button">
  <!---->
  <!---->
  <!----></button>
`;
